thirtyinch/build.gradle

```diff
@@ -36,14 +36,14 @@ configurations {
 }
 
 dependencies {
-    compileOnly "com.android.support:appcompat-v7:$supportLibraryVersion"
+    compileOnly 'androidx.appcompat:appcompat:1.0.0'
 
-    testCompileOnly "com.android.support:support-annotations:$supportLibraryVersion"
+    testCompileOnly 'androidx.annotation:annotation:1.0.0'
     testImplementation "junit:junit:$junitVersion"
     testImplementation "org.mockito:mockito-core:$mockitoVersion"
     testImplementation "org.assertj:assertj-core:$assertjVersion"
 
-    androidTestImplementation "com.android.support.test:runner:$supportTestVersion"
+    androidTestImplementation 'androidx.test:runner:1.1.0-beta02'
```

Comment:
it seems there is already a stable version:
https://mvnrepository.com/artifact/androidx.test/runner/1.1.0